packages/charting/src/components/PieChart/Arc/Arc.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as shape from 'd3-shape';
+import { IArcProps, IArcStyles } from './Arc.types';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { getStyles } from './Arc.styles';
+
+export class Arc extends React.Component<IArcProps, {}> {
+  // tslint:disable:no-any
+  public arc: any;
```

Comment:
I see you've added types for D3 in this PR, which is really good. 👍 Is there a reason you can't use them here?